pages/project-version/update/js/index.js

```diff
@@ -34,12 +34,13 @@ start({
     canTransferDraft: state.static.canTransferDraft,
     transferInProgress: state.static.transferInProgress,
     establishments: state.static.user.asruUser
-      ? state.static.project.licenceHolder.establishments
+      ? state.model.licenceHolder.establishments
       : state.static.user.establishments.map(e => ({ id: e.id, name: e.name })),
     showConditions: state.static.showConditions,
     editConditions: state.static.editConditions,
     raCompulsory: state.model.raCompulsory,
-    training: state.static.training
+    training: state.static.training,
+    licenceHolder: state.static.licenceHolder
```

Comment:
It appears from this line combined with the change at L37 that the licence holder is at `state.static.licenceHolder` _and_ `state.model.licenceHolder`.

Is there a meaningful difference between those two things?

Reply:
If there isn't then I'd have a strong preference to remove the one from `static` and one serve the value once to avoid unnecessary data being send in the page payload.

Reply:
I'm looking at this atm